libopenage/testing/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 add_sources(libopenage
-	testing.cpp
+	"testing.cpp"
+	"benchmark_test.cpp"
```

Comment:
:D they make sense if there are variables involved, but for consistency reasons we should skip them in add_sources like the rest of the project does.